MovieApp/MovieApp/App/CoreUI/ImageEnum.swift

```diff
@@ -0,0 +1,23 @@
+import UIKit
+
+enum ImageEnum {
+    
+    case homeTabBarItem
+    case homeTabBarItemSelected
+    case favouritesTabBarItem
+    case favouritesTabBarItemSelected
+    
+    var image: UIImage? {
+        switch self {
+        case .homeTabBarItem:
+            return UIImage(named: "HomeButton2.pdf")
+        case .homeTabBarItemSelected:
+            return UIImage(named: "HomeButton.pdf")
+        case .favouritesTabBarItem:
+            return UIImage(named: "FavButton2.pdf")
+        case .favouritesTabBarItemSelected:
+            return UIImage(named: "FavButton.pdf")
+        }
+    }
```

Comment:
Also, we're using an extension:
```swift
import UIKit

public extension UIImage {

    convenience init(with bundleImage: BundleImage) {
        self.init(named: bundleImage.rawValue)!
    }

}
```

This enables us to separate the enum that holds the image information and the actual UIImage itself. That's useful when passing images through layers since you'd want to avoid importing UIKit anywhere but in the UI layer.

Reply:
Implemented this [here](https://github.com/lihter/Movie-App/pull/24/files#diff-c800cd7d79afa1a867a8b6759952344793bb31cc9f6dc3ea4cae2db6342fc1d3R3) and I've also deleted my `var image: UIImage?` from `ImageEnum/BundleImage`